### PR TITLE
Update RVM and add rubygems deploy target

### PR DESCRIPTION
**Pull request by mcinerney**
* Update RVM before_install to ensure we build against the latest ruby versions as defined in the .travis.yml
* Add rubygems deployment - on tag push

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,17 @@
 language: ruby
 rvm:
-  - 2.0
-  - 2.1
-  - 2.2
-
+- 2.0
+- 2.1
+- 2.2
```
Comment (gingermusketeer) [inline .travis.yml]: I think these should be indented

Comment (mcinerney) [inline .travis.yml]: I thought that too, but whenever you use the Travis CLI, it changes it to this. So I let it win to stop the continual struggle

On 6 May 2015 03:09, Max Brosnahan <notifications@github.com> wrote:

In .travis.yml<https://github.com/westfieldlabs/apivore/pull/41#discussion_r29690105>:

> @@ -1,8 +1,17 @@
>  language: ruby
>  rvm:
> -  - 2.0
> -  - 2.1
> -  - 2.2
> -
> +- 2.0
> +- 2.1
> +- 2.2


I think these should be indented

Comment (gingermusketeer) [inline .travis.yml]: Ah right. Weird

**Comment by gingermusketeer** [comment]
LGTM

**Pull request merged**